wire-ios/Wire-iOS/Sources/UserInterface/Helpers/KeyboardHeight.swift

```diff
@@ -32,12 +32,19 @@ final class KeyboardHeight: NSObject {
     }
 
     private static func phoneKeyboardHeight() -> CGFloat {
+        let bottomInset = getSafeAreaBottomInset() ?? 0
         switch UIScreen.main.bounds.height {
         case 667: return 258
         case 736: return 271
-        case 812: return 253 + UIScreen.safeArea.bottom
+        case 812: return 253 + bottomInset
```

Comment:
We should get rid of this code eventually.

Reply:
Agreed!